{{cookiecutter.project_slug}}/stylelint.config.js

```diff
@@ -0,0 +1,98 @@
+module.exports = {
```

Comment:
We have a https://github.com/team-rawbot/stylelint-config-rawbot preset you can extend from, just to add the tailwind directives.

Reply:
Fine for me but is it not a problematic that styelint config rawbot use by default the stylelint scss plugin https://github.com/team-rawbot/stylelint-config-rawbot/blob/master/index.js#L3 ?

Reply:
Ha, didn't think about that, we didn't really take care of Stylelint since a while… I think we should release an update that matches with Tailwind.

Reply:
I've released stylelint-config-rawbot 1.0.0 without Scss plugin